todo_app/app.py

```diff
@@ -1,14 +1,41 @@
-from flask import Flask
+
+from flask import Flask,request, render_template,redirect,url_for
+
 
 from todo_app.flask_config import Config
+from todo_app.data.session_items import get_items
+from todo_app.data.session_items import add_item
+from todo_app.data.session_items import get_item
+from todo_app.data.session_items import save_item
```

Comment:
Can be simplified to `from todo_app.data.session_items import get_items, add_item` etc